backend/app/views/backoffice/users/index.html.erb

```diff
@@ -21,16 +21,25 @@
       <th>
         <%= sort_link @q, :last_sign_in_at, t("backoffice.users.index.last_sign_in_at") %>
       </th>
+      <th>
+      </th>
+      <th class="opacity-25"><%= svg "menu" %></th>
     </tr>
   </thead>
   <tbody>
     <% @users.each do |user| %>
       <tr>
         <td><%= user.full_name %></td>
         <td><%= user.email %></td>
-        <td><%= user.account&.name %></td>
+        <td><%= account_link_for user.account %></td>
         <td><%= I18n.l user.created_at.to_date %></td>
         <td><%= I18n.l user.last_sign_in_at&.to_date, default: '' %></td>
+        <td><%= link_to t("backoffice.common.edit"), edit_backoffice_user_path(user.id), class: "link-button" %></td>
+        <td>
+          <%= render "menu" do %>
+            <%= render "backoffice/base/menu/delete", path: backoffice_user_path(user), model: t("backoffice.common.user") %>
```

Comment:
Would not be better to have common `menu_item` partial and write this as 

```
<%= render "menu-item %>
  <%= link_to t("backoffice.common.delete"), backoffice_user_path(user),
       data: {
         turbo_method: :delete,
         turbo_confirm: t("backoffice.messages.delete_confirmation", model: model.downcase)
       } %>
<% end %>
```

We need to put Reject and Approve also inside that menu.